src/CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+
+_⚠️ Please note: you may find it easier to generate diffs and make edits to the tutorial by opening `/dist/code/super-rentals/code` in a separate code editor window._
+
+Once you have put a `run: pause` at the spot in your prose where the diff needs to be captured, you can navigate to `/dist/code/super-rentals/code` in the terminal. First, check that the diff is included in the current changes:
+
+```
+➜  code git:(your-branch-name) ✗ git diff
+```
+
+Next, save your diff to some temporary file that you will open in a code editor in a moment (`my_awesome_patch` in the example below). You want to be sure to generate the diff specifically with only 1 line of context. You can do this by specifying `git diff --unified=1` or `git diff -U1`.
+
+```
+➜  code git:(your-branch-name) ✗ git diff -U1 > my_awesome_patch
+➜  code git:(your-branch-name) ✗ code my_awesome_patch
+```
+
+Once you have opened your temporary file containing your diff, edit it so that the only thing surrounding the diff is the line number additions subtractions.
+
+    ```
+    @@ -9,2 +9,3 @@
+    Router.map(function() {
+    +  this.route('about');
+    });
+    ```
+
+Finally, wrap the newly-modified diff content in the `run:file:patch` command. Be sure to specify the language of the file, the filename, and the `cwd=super-rentals` option so that the patch is applied in the app's directory.
+
+For example:
+
+    ```run:file:patch lang=js cwd=super-rentals filename=app/router.js
+    @@ -9,2 +9,3 @@
+    Router.map(function() {
+    +  this.route('about');
+    });
+    ```
+
+Once you have added the diff to the prose using the `run:file:patch` command, you can remove the `run:pause` command. In order to test that your patch was successfully applied, be sure to:
+
+1. Run `git checkout` any changes in the workoing directory of `/dist/code/super-rentals/code`.
+2. Regenerate your modified chapter (`yarn generate src/chapters/00-your-chapter-to-edit-here.md`).
+
+Finally, you can look at your modified chapter in `/dist/chapters` and make sure that your patch renders as you expect in the markdown!
+
+#### Adding screenshots
+
+TODO: add content!
+
+#### Adding gifs
+
+TODO: add content!
+
+#### Saving a chapter
+
+Once a chapter is ready to be published and files, diffs, and screenshots have all been added as necessary, the last step is to commit the code changes at the end of the chapter. This is done usiing the `run:checkpoint` command.
+
+For example:
+
+    ```run:checkpoint cwd=super-rentals
+    Chapter 3
+    ```
+
+This should always be the last thing at the end of every chapter, as it will lint and run tests against the code generated in the chapter. The commit message will be whatever content is in the code block. We prefer the chapter title as the content of the commit.
+
+Before opening a pull request, be sure to check the `/dist/chapters` to make sure that the markdown generated by your changes to a chapter still looks correct.
```

Comment:
After the pull request is merged into the master branch, the markdown/code output will be pushed to a branch on the guides and super rentals repos (link to repo/branches?) so they can be further reviewed by their maintainers before integrating